src/figuresClasses.ts

```diff
@@ -1,19 +1,85 @@
 export interface Figure {
-
+  shape: string,
+  color: string,
```

Comment:
let's introduce typings for `shape` and `color`. try to use `enum`

Reply:
I'm not sure but tests wouldn't pass with Colors enum because data is string 'green' instead of Colors.green